composer.json

```diff
@@ -51,6 +51,24 @@
             "tests/views"
         ]
     },
+    "scripts": {
+        "test": "phpunit -c . tests/",
+        "lint": "parallel-lint --exclude vendor .",
+        "smell": "phpcs --standard=tools/codesniffer/JoindInPSR2/ruleset.xml --ignore=**/config.php,**/database.php,vendor,tools,tests/bootstrap.php --extensions=php --report-checkstyle=build/logs/checkstyle.xml --runtime-set ignore_warnings_on_exit true -p .",
+        "security": "security-checker security:check composer.lock",
+        "diff": [
```

Comment:
I would call this task coverage instead of diff

Reply:
maybe filter?   The thing is it's not generating coverage, phpunit is already doing that via the dist file.